database/database.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+import boto3
+import os
+
+db_name = os.environ['DB_NAME']
+username = os.environ['DB_USERNAME']
+password = os.environ['DB_PASSWORD']
+
+# Get database instance endpoint
+client = boto3.client('rds')
+instances = client.describe_db_instances()
+
+for db in instances["DBInstances"]:
+    if db["DBName"] == db_name:
+        apsidb = db
```

Comment:
nie wiem, czy inicjalizacja zmiennej wewnątrz pętli jest elegancka, w Pythonie to działa, no ale nigdy nie byłe do tego jakoś przekonany. Jaka jest Twoja opinia na ten temat?

Reply:
Też mi się tak wydaje, poprawione.